src/index.css

```diff
@@ -6,20 +6,18 @@
 }
 
 body {
-  margin: 0;
+  display: flex;
```

Comment:
- Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/).
- [ ] Make sure the main page is aligned in the center, you need to fix the margin-left and margin-right 👍🏻
- [ ] The footer should have the total width of his parent container, kindly address this and make sure  Book Title,  Author, and Category input match the original design and the Add Book as well look 👇🏻 

- Your Design:

!
![im1](https://user-images.githubusercontent.com/79058364/211125235-5b6e18d3-df6d-4343-a6c2-27c09e38251f.png)


- Original Design:

![image](https://user-images.githubusercontent.com/79058364/191857667-6f501420-6d4a-41ab-a615-f0829889a246.png)